agent/eventhandler/handler_test.go

```diff
@@ -28,10 +28,10 @@ import (
 )
 
 func contEvent(arn string) api.ContainerStateChange {
-	return api.ContainerStateChange{TaskArn: arn, ContainerName: "containerName", Status: api.ContainerRunning}
+	return api.ContainerStateChange{TaskArn: arn, ContainerName: "containerName", Status: api.ContainerRunning, Container: &api.Container{}}
```

Comment:
`Container.GetSentStatus()` and `Container.SetSentStatus()` are called without `nil`-checks.